client/index.html

```diff
@@ -261,14 +267,14 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
-									<% themes.forEach(function(themeName) { %>
-										<option value="<%= themeName %>">
-											<%= themeName.charAt(0).toUpperCase() + themeName.slice(1) %>
+									{{#each themes}}
+										<option value="{{this}}">
+											{{firstLetterUppercase this}}
```

Comment:
We can do it on the server, but that does mean completely chaning the data structure for themes, as we need the lowercase ones as well.

It would go from ["default","crypto"] to [{"name": "Default", fileName: "default"},{"name": "Crypto", fileName: "crypto"}]

Are you both happy with this?